prereq_flowchart/majorinfo.py

```diff
@@ -0,0 +1,43 @@
+from selenium.webdriver.common.by import By
+import re
+import json
+import time
+from selenium import webdriver
+from selenium.common.exceptions import StaleElementReferenceException
+
+file=open("majorRequirements.json","w")
```

Comment:
all of this code should be inside a `if __name__ == "__main__":` block so that it doesn't have side effects if we try to import it